contents/go/Go.md

```diff
@@ -0,0 +1,261 @@
+# Introduction to Go
+
+Author(s): [Cara Leong](https://github.com/craaaa)
```

Comment:
Can add a table of contents? a ToC makes it easier to see the high-level structure.